src/Tests/Tests/Framework/EndpointTests/CrudTestBase.cs

```diff
@@ -164,21 +164,43 @@ protected CrudTestBase(TCluster cluster, EndpointUsage usage)
 				if (TestClient.Configuration.RunIntegrationTests)
 				{
 					this.IntegrationSetup(client);
-					this._usage.CalledSetup = true;
+					_usage.CalledSetup = true;
 				}
 
-				var sf = Sanitize(RandomFluent);
-				dict.Add(Integration.ClientMethod.Fluent, fluent(sf, client, f => fluentBody(sf, f)));
+				var sf = this.Sanitize(RandomFluent);
+				dict.Add(ClientMethod.Fluent, fluent(sf, client, f => fluentBody(sf, f)));
 				if (this.TestOnlyOneMethod) return dict;
 
-				var sfa = Sanitize(RandomFluentAsync);
-				dict.Add(Integration.ClientMethod.FluentAsync, await fluentAsync(sfa, client, f => fluentBody(sfa, f)));
+				var sfa = this.Sanitize(RandomFluentAsync);
+				dict.Add(ClientMethod.FluentAsync, await fluentAsync(sfa, client, f => fluentBody(sfa, f)));
 
-				var si = Sanitize(RandomInitializer);
-				dict.Add(Integration.ClientMethod.Initializer, request(si, client, initializerBody(si)));
+				var si = this.Sanitize(RandomInitializer);
```

Comment:
I think we ought to pass across the codebase for 7.x and standardize the conventions 😄